extensions/wikia/PortableInfobox/js/PortableInfobox.js

```diff
@@ -25,22 +25,46 @@
 	};
 
 	var CollapsibleGroup = {
-		init: function($content) {
+		init: function ($content) {
 			var $collapsibleGroups = $content.find('.pi-collapse');
 
-			$collapsibleGroups.each( function( index, group ) {
+			$collapsibleGroups.each(function (index, group) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "group". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)